web/packages/build/package.json

```diff
@@ -31,9 +31,9 @@
     "jest-environment-jsdom": "^29.7.0",
     "jest-fail-on-console": "^3.3.0",
     "jest-styled-components": "^7.2.0",
-    "jsdom": "^21.1.0",
+    "jsdom": "^24.1.1",
     "rollup-plugin-visualizer": "^5.12.0",
-    "typescript-eslint": "^7.14.1",
+    "typescript-eslint": "^8.0.0",
```

Comment:
https://github.com/gravitational/teleport/pull/44930 should probably be rolled into this PR as well, shouldn't it?

Reply:
Yes. Sorry for confusion, this PR is still wip (I probably should have put it in draft). Eslint requires a bit bigger of an update because the config files change completely. Will ping again when it's ready